lms/djangoapps/teams/models.py

```diff
@@ -0,0 +1,98 @@
+"""
+Django models related to teams functionality.
+"""
+
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils.translation import ugettext_lazy
+from django.conf import settings
+from django_countries.fields import CountryField
+from .utils import slugify
+
+from xmodule_django.models import CourseKeyField
+
+
+class CourseTeam(models.Model):
+    """
+    This model represents team related info.
+    """
+
+    team_id = models.CharField(max_length=255, unique=True)
+    name = models.CharField(max_length=255)
+    is_active = models.BooleanField(default=True)
+    course_id = CourseKeyField(max_length=255, db_index=True)
+    topic_id = models.CharField(max_length=255, db_index=True, blank=True)
+    date_created = models.DateTimeField(auto_now_add=True)
+    # last_activity is computed through a query
+    description = models.CharField(max_length=1000)
+    country = CountryField(blank=True)
+    language = models.CharField(
```

Comment:
Nit: could we introduce a LanguageField that we can share here and with language proficiency? It would be great if we didn't have to duplicate the choices, help text etc.